packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -580,3 +580,71 @@ class _UiKitPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// This widget provides the minimal required functionality for a platform view.
```

Comment:
maybe the summary should say: 'Embeds an existing platform view in the widget tree`